helpers/resourceHints.js

```diff
@@ -16,28 +15,28 @@ const factory = globals => {
             return `<link rel="dns-prefetch preconnect" href="${host}" crossorigin>`;
         }
 
-        var hosts = [];
-
         // Add cdn
         const siteSettings = globals.getSiteSettings();
-        const cdnUrl = siteSettings['cdn_url'] || '';
-        if (cdnUrl != '') {
-            hosts.push(cdnUrl);
-        }
+        const hosts = siteSettings['cdn_url'] ?  [siteSettings['cdn_url']] : [];
 
         // Add font providers
-        const fontProviders = _.keys(getFonts('providerLists', globals.getThemeSettings(), globals.handlebars));
-        _.each(fontProviders, function(provider) {
-            if (typeof fontResources[provider] !== 'undefined') {
-                hosts = hosts.concat(fontResources[provider]);
+        const providers = Object.keys(getFonts('providerLists', globals.getThemeSettings(), globals.handlebars));
+
+        for (let i = 0; i < providers.length; i++) {
+            if (typeof fontResources[providers[i]] !== 'undefined') {
+                hosts.push(...fontResources[providers[i]]);
```

Comment:
Why not `hosts.concat`?

Reply:
Mainly to make hosts a const.

Reply:
It returns a new array too so I think it's doing more work.